api/test/e2e/base.go

```diff
@@ -54,6 +54,7 @@ func init() {
 
 	url := ManagerAPIHost + "/apisix/admin/user/login"
 	req, err := http.NewRequest(http.MethodPost, url, bytes.NewBuffer(requestBody))
+	req.Header.Add("Content-Type", "application/json")
```

Comment:
Just to make sure that FE will add this header when query the BE.

@nic-chen @juzhiyuan

Reply:
ya